### ISSUE TITLE
On Screen Keyboard & Multitouch

### ISSUE THREAD

**Issue by XDjackieXD** [papyros/papyros#29]
It would be really nice to have a fancy on screen keyboard and support for multitouch drag scroll, pinch zoom, ...
Papyros would be really cool for tablets/convertibles with this features.
##

**Comment by SuicSoft** [comment]
Yes, but I don't think the developer has a touch device.I don't have one too (just a normal laptop) and I think that each touch screen is different and their might not be drivers (software) for Linux.Just thinking if Papyros could be run on ARM tablets such the Nexus 9 (mine runs normal Android M) , 7 and 10

**Comment by XDjackieXD** [comment]
Most touch devices have Linux drivers already. I for myself have two devices with wacom multitouch digitizers (a Lifebook t730 which I can't get papyros to run on without an already running x server and a cintiq 13HD touch).
Many older touch-enabled devices (like the t730) are very cheap to get on ebay or simmilar. I got mine for 200€ with i5, 8GB RAM and an 120GB SSD...
Also to my knowledge Wayland already handles all the needed hardware abstraction like xinput (including multitouch support).

**Comment by SuicSoft** [comment]
I have seen ways to run Ubuntu on the Nexus 9 (http://forum.xda-developers.com/nexus-9/general/guide-linux-nexus-9-t2985958) and 7 (https://wiki.ubuntu.com/Nexus7). So maybe we can run Papyros Shell on these devices?

**Comment by lyonz** [comment]
http://doc.qt.io/QtVirtualKeyboard/qtvirtualkeyboard-enterprise-virtualkeyboard-virtualkeyboard-example.html
